docs/en/observability/profiling-troubleshooting.asciidoc

```diff
@@ -0,0 +1,235 @@
+Different hardening methodologies can impair host-agent operations and may, for example, result in pods continuously restarting after displaying a `CrashLoopBackoff` status.
+
+[discrete]
+[[profiling-kubernetes-podsecuritypolicy]]
+==== {k8s} PodSecurityPolicy (https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/[deprecated])
+
+This {k8s} API has been deprecated, but some still use it. A PodSecurityPolicy (PSP) may explicitly prevent the execution of `privileged` containers across the entire cluster.
+
+Since host-agent _needs_ privileges in most kernels/CRI, you need to build a PSP to allow the host-agent DaemonSet to run.
+
+[discrete]
+[[profiling-policy-engines]]
+==== {k8s} policy engines
+
+Read more about {k8s} policy engines in the https://github.com/kubernetes/sig-security/blob/main/sig-security-docs/papers/policy/kubernetes-policy-management.md[SIG-Security documentation].
+
+The following tools _may_ prevent the execution of host-agent pods as the Helm chart builds a cluster role and binds it into the host-agent service account (we use it for container metadata):
+
+* Open Policy Agent Gatekeeper
+* Kyverno
+* Fairwinds Polaris
+
+If you have a policy engine in place, configure it to allow the host-agent execution and RBAC configs.
+
+[discrete]
+[[profiling-network-config]]
+==== Network configurations
+
+In some instances, your host-agent pods may be running fine, but they will not connect to the remote data collector gRPC interface and stay in the startup phase, while trying to connect periodically.
+
+The following are potential causes:
+
+* {k8s} https://kubernetes.io/docs/concepts/services-networking/network-policies/[`NetworkPolicies`] define connectivity rules that prevent all outgoing traffic unless explicitly allow-listed.
+* Cloud or datacenter provider network rules are restricting egress traffic to allowed destinations only (ACLs).
+
+[discrete]
+[[profiling-os-level-security]]
+==== OS-level security
+
+These settings _are not part of {k8s}_ and may have been included in the node setup. They can prevent the host-agent from working properly, as they intercept syscalls from the host-agent to the kernel and modify or block them.
+
+If you have implemented security hardening (some providers listed below), you should know the privileges the host-agent needs.
+
+* gVisor on GKE
+* seccomp filters
+* AppArmor LSM
+
+
+[discrete]
+[[profiling-submit-support]]
+== Submit a support request
+
+You can submit a support request from the https://cloud.elastic.co/support[support request page] in the {ecloud} console.
+
+In the support request, specify if your issue deals with the host-agent or the Kibana app.
+
+[discrete]
+[[profiling-send-feedback]]
+== Send feedback
+
+If troubleshooting and support are not fixing your issues, or you have any other feedback that you want to share about the

```

Comment:
This is probably a question for @iogbole: Do we want to encourage people to send us an email to `profiling-feedback@elastic.co`? Usually we direct people to [our Discuss forum](https://discuss.elastic.co/) and I noticed there is already a [category for profiling](https://discuss.elastic.co/c/observability/profiling/92). Do we want to use this instead once the beta opens?

Reply:
Good question, @danielmitterdorfer.  The email still applies in 8.7.  We recently created the profiling category in preparation for public beta.